tests/xtimer_hang/main.c

```diff
@@ -38,14 +43,31 @@ char stack_timer2[TEST_TIMER_STACKSIZE];
 
 void* timer_func(void* arg)
 {
+#ifdef WORKER_THREAD_PIN
+    printf("Debug worker thread port 0x%02x pin %d\n",  WORKER_THREAD_PORT, WORKER_THREAD_PIN);
+    gpio_t worker_pin = GPIO_PIN(WORKER_THREAD_PORT, WORKER_THREAD_PIN);
```

Comment:
Instead of

```c
gpio_t worker_pin = GPIO_PIN(WORKER_THREAD_PORT, WORKER_THREAD_PIN);
```

why not 

```
gpio_t worker_pin = WORKER_THREAD_PIN;
```

and define 

```
-DWORKER_THREAD_PIN=GPIO_PIN\(PORTF, 6\)
```

in the Makefile? We do that at [several](https://github.com/RIOT-OS/RIOT/blob/master/tests/pkg_u8g2/Makefile#L19) [places](https://github.com/RIOT-OS/RIOT/blob/master/tests/driver_nrf24l01p_lowlevel/Makefile#L14).

(@kYc0o what do you think?)

Reply:
+1 on defining `WORKER_THREAD_PIN` and also `MAIN_THREAD_PIN`